source/core/slang-platform.cpp

```diff
@@ -331,4 +336,24 @@ static const PlatformFlags s_familyFlags[int(PlatformFamily::CountOf)] = {
 #endif
 }
 
+/* static */ void PlatformUtil::backtrace(uint32_t uid)
```

Comment:
`uid` and `fprintf(stderr, "IR instruction UID %u created at:\n", uid);` should not be part of this function.